src/api-guides/e-receipts.markdown

```diff
@@ -1,173 +1,159 @@
 ---
-title: eReceipts Integration Guide
+title: E-Receipts Integration Guide
 layout: reference
 ---
 
+At SAP Concur, e-receipts are digital representations of receipts with both a receipt image and structured data. The e-receipts come from pre-authorized and trusted merchants and are transmitted to SAP Concur using our **Receipt** API (not email) on behalf of individual users who have connected their accounts/profiles between SAP Concur and the merchant in advance.
 
+## Getting Started
 
-## eReceipts Guide
+**New App Center Partners:**  Your Company must have entered into a commercial agreement with SAP Concur. If you have not yet entered into said agreement with SAP Concur, click [here](mailto:concur_bizdev@sap.com) to contact our business development team.
 
+**Existing App Center Partners:**  The SAP Concur Platform team will need to configure your existing sandbox, among other things to enable access.  You can make that request by contacting your [Alliance Manager](mailto:concur_AppCenterAlliance@sap.com?subject=Receipts%20v4%20Migration%20Request).
 
-At Concur, e-receipts are digital representations of receipts with both a receipt image and structured data. The e-receipts come from pre-authorized and trusted merchants and are transmitted to Concur using our **Receipt** API (not email) on behalf of individual users who have connected their accounts/profiles between Concur and the merchant in advance.
+## User Connections / Authentication
 
-### Getting Started
+The below Authentication Methods are available to obtain an access token. Password Grant or Authorization Grant or One-Time Password Grant is required for Certification:
 
-**New App Center Partners:**  Your Company must have entered into a commercial agreement with Concur.  If you have not yet entered into said agreement with Concur, click [here](/why-concur.html) to contact our business development team
+- [Password Grant](/api-reference/authentication/apidoc.html#password_grant) - Connect from the SAP Concur App Center.
+- [Authorization Grant](/api-reference/authentication/apidoc.html#auth_grant) - Connect from the Partner App.
```

Comment:
Remove the underscore and replace with hyphen in the URI -- I will make the corresponding anchor element change in the referred pages. Note: Changing the fragments (...#whatever) doesn't break the path before the fragment.